tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1106,25 +1108,33 @@ handle_build() {
 
 		echo "${ARTEFACT_REGISTRY_PASSWORD}" | sudo oras login "${ARTEFACT_REGISTRY}" -u "${ARTEFACT_REGISTRY_USERNAME}" --password-stdin
 
-		case ${build_target} in
-			kernel*-confidential)
-				sudo oras push \
-					${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) \
-					${final_tarball_name} \
-					"kata-static-${build_target}-modules.tar.xz" \
-					${build_target}-version \
-					${build_target}-builder-image-version \
-					${build_target}-sha256sum
-				;;
-			*)
-				sudo oras push \
-					${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) \
-					${final_tarball_name} \
-					${build_target}-version \
-					${build_target}-builder-image-version \
-					${build_target}-sha256sum
-				;;
-		esac
+		tags=(latest-${TARGET_BRANCH}-$(uname -m))
+		if [ -n "${artefact_tag}" ]; then
+			tags+=("${artefact_tag}")
+		fi
+		# TODO - Ideally we would also tag release version if we are doing a release. How can we tell here?
```

Comment:
We bump the `${repo_root_dir}/VERSION` file before the release is tagged, meaning that if we know that we're doing a release, heeeey, we easily can get the version.

Now, how can we know that we're in a release?
I'd simply export an env var as part of the RELEASE workflow, and check for this env var here, adding then the version coming from `${repo_root_dir}/VERSION` as part of the tags.

WDYT?

Reply:
Yeah, that's worth a try. Thanks

Reply:
An attempt was made